### PR TITLE
Feat: Expose ping method from SQLx

### PR DESCRIPTION
**Pull request by ProbablyClem**
## PR Info

- Closes [Expose ping method from SQLx](https://github.com/SeaQL/sea-orm/issues/1578)

- Dependencies:
  - None

- Dependents:
  - None

## New Features

- [x] Added an `async fn ping(&self) -> Result<(), DbErr>`  to `DatabaseConnection`
- [x] Added an `async fn ping(&self) -> Result<(), DbErr>`  to `DatabaseTransaction
`
## More
Thoses methods are only wrappers for sqlx::Connection::ping as asked in the issue

It's my first PR on this repo so any feedback is welcomed !
Thanks you for reading

### CODE REVIEW

File: src/database/mock.rs [hunk 1]
```diff
@@ -203,6 +203,14 @@ impl MockDatabaseTrait for MockDatabase {
...
         self.db_backend
     }
+
+    fn ping(&self) -> Result<(), DbErr> {
+        match self.query_results.first() {
+            Some(Ok(_)) => Ok(()),
+            Some(Err(_)) => Err(conn_err("Disconnected")),
+            None => Err(query_err("`query_results` buffer is empty.")),
+        }
+    }
```
Comment (billy1624) [inline src/database/mock.rs:213]: Since mock connection is always available. We should simply return `Ok(())` here.

**Review by billy1624 — APPROVED** [review]
Thanks!! @ProbablyClem

File: src/driver/sqlx_sqlite.rs [hunk 2]
```diff
@@ -229,6 +229,19 @@ impl SqlxSqlitePoolConnection {
         self.metric_callback = Some(Arc::new(callback));
     }
 
+    /// Checks if a connection to the database is still valid.
```
Comment (tyt2y3) [inline src/driver/sqlx_sqlite.rs:232]:
```suggestion
```

**Review by tyt2y3 — COMMENTED** [review]
I don't think adding the method to ConnectionTrait is the best place.
Can we simply follow https://github.com/SeaQL/sea-orm/pull/1236 ?
Or is there a reason that makes that approach non-applicable in some places?

**Comment by billy1624** [comment]
@tyt2y3 please check again

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Review by tyt2y3 — APPROVED** [review]

**Comment by tyt2y3** [comment]
Sorry, my poor internet again. And seemingly there isn't a debounce.

**Pull request merged**